apps/extension/src/core/llm/cohere.ts

```diff
@@ -32,6 +32,8 @@ export function init(
       isStreamable: false,
       modelProvider: "cohere",
       defaultBaseUrl: "https://api.cohere.ai",
+      promptCost: 0, // TODO: Replace this with the correct price per token for cohere
+      completionCost: 0, // TODO: Replace this with the correct price per token for cohere
```

Comment:
@alexanderatallah Do you know where I can find the cost per token for Cohere? The pricing on their website seems to be based on something different from tokens: https://cohere.com/pricing